website/src/releases/2020-10/migrating-to-v17.mdx

```diff
@@ -1,19 +1,16 @@
 ---
 date: 2020-10-12
-title: Migrating to the new Apollo Client 3.0
+title: Migrating to v17
```

Comment:
I've renamed the release note, as it's about migrating to v17 now (which of course includes the apollo v3 migration).